src/views/ui_fungible_asset.py

```diff
@@ -250,7 +247,7 @@ def show_assets(self):
         )
         self.address_header.setText(
             QCoreApplication.translate(
-                'iris_wallet_desktop', 'address', None,
+                'iris_wallet_desktop', 'asset_id', None,
```

Comment:
here and in other calls to `QCoreApplication.translate` shouldn't be better to define a constant for the `iris_wallet_desktop` string?